packages/frontend/app/services/channels/-utils/channel-factory.ts

```diff
@@ -0,0 +1,64 @@
+import Channel from 'emberclear/models/channel';
+import Vote from 'emberclear/models/vote';
+import VoteChain from 'emberclear/models/vote-chain';
+import Identity from 'emberclear/models/identity';
+import { toHex } from 'emberclear/utils/string-encoding';
+import ChannelContextChain from 'emberclear/models/channel-context-chain';
+
+export function buildChannelInfo(channel: Channel): StandardMessage['channelInfo'] {
+  return {
+    uid: channel.id,
+    admin: buildChannelMember(channel.admin),
```

Comment:
I don't remember, but is the admin included in the hashing / construction of the chain?

Reply:
Yes. So the Channel model has admin and members, however they are mainly for ease of access. The contextChain field of the Channel model has the most updated context, the vote supporting it, and the previous chain.